Docs release notes July 26 2024 (#18151)
newrelic/docs-website: src/content/docs/release-notes/docs-release-notes/docs-7-26-2024.mdx

* In [infrastructure inventory UI page](/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure), added note that APM metadata is created in the ingest pipeline when infrastructure and APM agents are running on the same host.
* In [import, export, add, and share dashboards and charts](/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data/#export-pdf-limitations) added a section describing some of the limitations when exporting a dashboard to PDF.
* Clarified reduce data ingest and metrics in [OpenTelemetry for Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-otel). Also, added a new diagram.
* Microsoft dropped support Windows Server 2012 and 2012 R2 on October 10, 2023. We've updated [requirements for the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent) to reflect that.
* Updated the video on [intro to logs in context](/docs/logs/logs-context/logs-in-context/).

> * Updated "do not translate" tags across our site to improve the translation accuracy of UI and non-UI text strings.

Reviewer: ```suggestion
  * Updated `<DoNotTranslate>` tags across our site to improve the translation accuracy of UI and non-UI text strings.
```

Reviewer: Maybe actually include the tag name instead of putting it in quotes

Author: @ally-sassman Isn't the tag `<DNT>` now?

Reviewer: Ah I see, so maybe specify what it was and what it is now (i.e., <DoNotTranslate> is now <DNT>)

This could be useful for contributors to know what DNT is now

Reviewer: But not a hard suggestion, feel free to discard this

> * Updated "do not translate" tags across our site to improve the translation accuracy of UI and non-UI text strings.

Author: ```suggestion
* Updated "do not translate" `<DNT>` tags across our site to improve the translation accuracy of UI and non-UI text strings.
```
